perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1206,10 +1207,16 @@ sub redirect_or_confirm_creation : Private {
                 to => [ [ $report->user->email, $report->name ] ],
             } );
         }
-        $c->log->info($report->user->id . ' was logged in, showing confirmation page for ' . $report->id);
-        $c->stash->{created_report} = 'loggedin';
-        $c->stash->{template} = 'tokens/confirm_problem.html';
-        return 1;
+        if ($c->user_exists && $c->user->has_body_permission_to('planned_reports')) {
```

Comment:
Not sure $c->user_exists is needed here, as we are logged in if we get here, but no harm in leaving it :)